src/plugins/xfconf/README.md

```diff
@@ -13,9 +13,38 @@
 
 This is a storage plugin to mount the xfconf configuration settings.
 
+## Xfconf Terminology
+
+### Property
+
+A property in xfconf is the same as a key in libelektra i.e. it has a name and can hold one or more values.
```

Comment:
A key only has one value (or none). Can you please give an example of how a property in xfconf would be represented in Elektra.

Reply:
Definitely, this was may fault. I corrected this with d1e489242be9b082f94879d404135708e550887b.